modules/sync-narration.md

```diff
@@ -0,0 +1,200 @@
+| `textRef`   |  The absolute of relative URL of the textual resource which participates to the synchronized narration | URL | At the top level only |
+| `audioRef`  |  The absolute of relative URL of the audio resource which participates to the synchronized narration | URL | At the top level only |
+| `narration` | A recursive array of synchronization objects | Array of Synchronization Item or Sub Narration | Yes |
+
+
+The Sub Narration object is defined as:
+
+| Key  | Definition | Format | Required? |
+| ---- | -----------| -------| ----------|
+| `role`      | Semantic information relative to the role of the object in the publication | [EPUB Structural Semantics](#references) | No |
+| `narration` | A recursive array of synchronization objects | Array of Synchronization Item or Sub Narration | Yes |
+
+
+The Synchronization Item object is defined as:
+
+| Key  | Definition | Format | Required? |
+| ---- | -----------| -------| ----------|
+| `text`  | The fragment identifier of the textual fragment which participates to the synchronized narration | URI Fragment Identifier | Yes |
+| `audio` | The fragment identifier of the audio fragment which participates to the synchronized narration | Media Fragment URI | Yes |
+
+**Notes:**
+
+Structural semantics offer a way to selectively filter out content based on semantics ("skippability" in the DAISY world)
+
+The recursive structure offers a way to jump out of complex structures and back into the reading flow ("escapability" in the DAISY world).
+
+In case we decide to extend the structure to image and video, using `image` and `video` would be consistent with the [latest work of the W3C CG](https://w3c.github.io/sync-media-pub/sync-media.html). 
+
+## Example
+
+```json
+{
+  "textRef": "/text/chapter1.html",
+  "audioRef": "/audio/chapter1.mp3",
+  "narration": [
+    {
+      "text": "#id1",
+      "audio": "#t=0.0,1.2"
+    },
+    {
+      "text": "#id2",
+      "audio": "#t=1.2,3.4"
+    },
+    {
+      "role": "footnote",
+      "text": "#id3",
+      "audio": "#t=3.4,5.6"
+    },
+    {
+      "role": "aside",
+      "narration": [
+        {
+          "text": "#id4",
+          "audio": "#t=5.6,7.8"
+        },
+        {
+          "text": "#id5",
+          "audio": "#t=7.8,9.1"
+        }
+      ]
+    },
+    {
+      "text": "#id6",
+      "audio": "#t=9.1,10.2"
+    }
+  ]
+}
+```
+
+## Mime Type and file extension
+
+This specification introduces a dedicated media type value to identify a Synchronized Narration document: `application/vnd.syncnarr+json`.
+
+When saved as a file, a Synchronized Narration document <strong class="rfc">should</strong> have the following extension: `.json`. 
+
+## Declaring Synchronized Narration documents in a Manifest
+
+Each Synchronized Narration document used in a publication <strong class="rfc">must</strong> be declared in the Readium Webpub Manifest as an `alternate` resource with the proper media type. 
```

Comment:
I always considered `alternate` to be an alternative rendition of a particular resources. The Synchronized Narration object seems more like an augmentation of the resource.

But maybe I'm wrong on the semantics of `alternate`?